### PR TITLE
164796889-linkedin-authentication

### PR DESCRIPTION
**Pull request by Oluwaseyi000**
#### Description
This feature allows users to signup and signin to Authors' Haven using Linkedin

#### Type of change
New feature (a non-breaking change which adds functionality)

#### How Has This Been Tested?
Integration testing
Unit testing

#### Checklist:
My code follows the style guidelines of this project
My changes generate no new warning

#### PT
[165294276](https://www.pivotaltracker.com/story/show/165294276)

### CODE REVIEW

File: src/controllers/authentication/user.js [hunk 1]
```diff
@@ -58,3 +59,56 @@ export const createUser = async (req, res) => {
...
+        email: linkedinEmail,
+        username: `${lastname} ${firstname}`,
+        imageUrl: LinkedinImageUrl,
+        password: linkedinId
+      }
+    });
+
+
+    const { id, fullName, email, username, imageUrl } = User.dataValues;
+    console.log(user.User);
```
Comment (ssanusi) [inline src/controllers/authentication/user.js]: @Oluwaseyi000 excellent work remove the console log

File: src/controllers/authentication/user.js [hunk 2]
```diff
@@ -58,3 +59,56 @@ export const createUser = async (req, res) => {
...
+
+    const token = Auth.generateToken({ id, fullName, email, username, imageUrl });
+
+    res.cookie('jwt-token', token);
+    res.redirect('/api/v1');
+  } catch (error) {
+    console.log(error);
```
Comment (ssanusi) [inline src/controllers/authentication/user.js]: @Oluwaseyi000 same as this too remove the console log

**Review by ssanusi — COMMENTED** [review]
@Oluwaseyi000 excellent work going out of your way to bring linkedin auth to the application remove the console logs thanks

File: src/controllers/authentication/user.js (hunk 1 above)
Comment (Oluwaseyi000) [reply src/controllers/authentication/user.js]: fixed

File: src/controllers/authentication/user.js (hunk 2 above)
Comment (Oluwaseyi000) [reply src/controllers/authentication/user.js]: fixed

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**